app/reducers/navigation/index.js

```diff
@@ -16,7 +16,8 @@ const initialState = {
     leftDrawerOpen: false,
     leftDrawerRoute: null,
     rightDrawerOpen: false,
-    rightDrawerRoute: null
+    rightDrawerRoute: null,
+    leftDrawerDisabled: false
```

Comment:
Can you make this into `drawersDisabled`? I can't think of any circumstances when you'd want one disabled and not the other